.github/workflows/main.yml

```diff
@@ -124,12 +139,17 @@ jobs:
         run:  |
           cd build
           make pinetime-mcuboot-app
+          echo ""
 
       - name: Create firmware image
```

Comment:
this step should be deleted -- `make pinetime-mcuboot-app` already does this (except the file name is `image` instead of `img`)